docs/source/tutorials/datalab.ipynb

```diff
@@ -0,0 +1,519 @@
+    "lab.report()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also increase the verbosity of the `report` to see additional information about the data issues and control how many top-ranked examples are shown for each issue."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lab.report(num_examples=10, verbosity=2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice how the number of flagged outlier issues has changed after specfying different settings to use for outlier detection."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding a custom IssueManager (advanced)\n",
+    "\n",
+    "`Datalab` detects pre-defined types of issues for you in one line of code: `find_issues()`. What if you want to check for other custom types of issues along with these pre-defined types, all within the same line of code?\n",
+    "\n",
+    "All issue types in `Datalab` are subclasses of cleanlab's `IssueManager` class.\n",
+    "To register a custom issue type for use with `Datalab`, simply also make it a subclass of `IssueManager`.\n",
+    "\n",
+    "The necessary members to implement in the subclass are:\n",
+    "\n",
+    "- A class variable called `issue_name` that acts as a unique identifier for the type of issue.\n",
+    "- An instance method called `find_issues` that:\n",
+    "  - Computes a quality score for each example in the dataset (between 0-1), in terms of how *unlikely* it is to be an issue.\n",
+    "  - Flags each example as an issue or not (may be based on thresholding the quality scores).\n",
+    "  - Combine these in a dataframe that is assigned to an `issues` attribute of the `IssueManager`.\n",
+    "  - Define a summary score for the overall quality of entire dataset, in terms of this type of issue. Set this score as part of the `summary` attribute of the `IssueManager`.\n",
+    "  \n",
+    "To demonstrate this, we create an arbitrary issue type that checks the divisibility of an example's index in the dataset by 13."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab.experimental.datalab.issue_manager import IssueManager\n",
+    "from cleanlab.experimental.datalab.factory import register\n",
+    "\n",
+    "\n",
+    "def scoring_function(idx: int) -> float:\n",
+    "    if idx == 0:\n",
+    "        # Zero excluded from the divisibility check, gets the highest score\n",
+    "        return 1\n",
+    "    rem = idx % 13\n",
+    "    inv_scale = idx // 13\n",
+    "    if rem == 0:\n",
+    "        return 0.5 * (1 - np.exp(-0.1*(inv_scale-1)))\n",
+    "    else:\n",
+    "        return 1 - 0.49 * (1 - np.exp(-inv_scale**0.5))*rem/13\n",
```

Comment:
Ref: this [comment thread](https://github.com/cleanlab/cleanlab/pull/614#discussion_r1128126905)

13 here is hard coded in the scoring function instead of allowing users to pass in alternate values of `div`.